Manual.md

```diff
@@ -0,0 +1,145 @@
+
+***
+
+## Introduction
+
+Each file under `packages/` is a recipe for `crew`, each recipe contains the instructions to compile from source and/or install as a binary package using `crew`. These package files are called `.rb`'s within this repository, but are otherwise know as ruby scripts.  
+
+The `.rbs`'s are ruby scripts called by `crew`<sup>(The Chromebrew package manager)</sup>. Each `.rb` must follow a explicit set of rules laid out by `crew`, these rules are provided within this Manual.
+
```

Comment:
This introduction could be worded a bit better to give a more general high level overview of the content.  The current introduction suggests it specifically addresses how to create and install packages.  While that is still helpful, it would probably make more sense if it mentioned an architectural overview of how Chromebrew works as a package manager which not only includes instructions for building and installing but also how to maintain packages such as updating and removing as well.  Maybe it would help clarify things a bit if you added a #Purpose section?

Reply:
I'll have to think of a way to do such, it may take me a little while, my creative energy drys up quickly during projects and I'll have to come up with what to put here.

Reply:
I've made some edits and hopefully they provide you with what you are looking for.

Reply:
```markdown
# Purpose
The purpose of this Manual is to properly document how `crew` works, create a proper rundown of how and what each function and/or variable does/works. Note that this Manual does not include documentation of functions and variables 
used in rare cases, such as `case` or `if`, as these are documented in the ruby programming language itself.  
Note that a understanding of `shell`, `ruby` and the Linux base system should exist before writing packages, as each package script uses such.

> `#`'s are used as placeholders/wild-cards within this Manual.

***

## Crew
`crew`<sup>Or Chromebrew</sup> is the native package manager for ChromeOS, it uses ruby scripts found in `/usr/local/lib/crew/packages`
to install each package. Each package script contains the instructions to build and install packages, how to maintain each package, and what should be done to remove each package. `crew` is an advnaced package manager that interacts with the Linux base system using the `shell` and `ruby` programming languages. `crew` is designed to implement a simple   means of compiling and installing packages with the ChromeOS distro. 
***
```